utils/readDir.js

```diff
@@ -0,0 +1,15 @@
+const fse = require('fs-extra');
+
+const { isAFileRE, shouldBeIgnoredRE } = require('./regEx.js');
+
+const excludedDirs = [
```

Comment:
Do we still need this now that we have gotten rid of `src/pages/search/`?

Reply:
Nope! Meant to talk about that in the PR that removed it, but forgot, sorry.